tasks/install_windows.yml

```diff
@@ -0,0 +1,63 @@
+---
+# File: install_remote.yml - package installation tasks for Consul
+
+- name: Validate remote Consul directory
+  win_file:
+    path: /tmp/consul
+    state: directory
+
+- name: Verify TLS1.2 is used
```

Comment:
Windows attempts to use TLS1.0 by default when downloading files, falling back to SSL3, and never trying TLS1.2. This is fixed in later versions of Windows.

This task sets a typical value in the registry just in case it isn't there, ensuring that TLS1.2 is tried first, *then* TLS1.0, *then* SSL3.